product-mini/platforms/linux-sgx/enclave-sample/Enclave/Enclave.cpp

```diff
@@ -118,17 +196,18 @@ handle_cmd_init_runtime(uint64 *args, uint32 argc)
     /* initialize runtime environment */
     if (!wasm_runtime_full_init(&init_args)) {
         LOG_ERROR("Init runtime environment failed.\n");
-        args[0] = false;
-        return;
+        ret = false;
```

Comment:
No need to set ret to false again since it is initialized as false.

Reply:
I keep `ret = false` to avoid further maintainer may initialize `ret` with other value and forget to modify here

Reply:
> No need to set ret to false again since it is initialized as false.

There are case developer forget to set it 
https://github.com/intel/ehsm/issues/265#issuecomment-1610803400

Reply:
OK, got it, thanks.